src/dbInterface.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const db = require('./db.js');
+
+const getAccount = login => {
+  for (const acc of db.accounts) {
+    if (acc.login === login) return acc;
+  }
+  return null;
+};
+
+const getBanks = accId => {
+  const banks = [];
+  for (const bank of db.banks) {
+    if (bank.account === accId) banks.push(bank);
+  }
+  return banks;
+};
+
+const getCards = accId => {
+  const banks = getBanks(accId);
+  const cards = [];
+  for (const card of db.cards) {
+    for (const bank of banks) {
+      if (card.bank === bank.id) cards.push(card);
+    }
+  }
```

Comment:
Maybe more functional style?

```suggestion
  const banks = getBanks(accId).map(bank => bank.id);
  return db.cards.filter(card => banks.includes(card.bank));
```